src/test/java/teammates/test/driver/HtmlHelper.java

```diff
@@ -36,7 +36,7 @@
     private static final String REGEX_BLOB_KEY = "(encoded_gs_key:)?[a-zA-Z0-9-_]{10,}";
     private static final String REGEX_QUESTION_ID = "[a-zA-Z0-9-_]{40,}";
     private static final String REGEX_COMMENT_ID = "[0-9]{16}";
-    private static final String REGEX_DISPLAY_TIME = "(0[0-9]|1[0-2]):[0-5][0-9] [AP]M( UTC)?";
+    private static final String REGEX_DISPLAY_TIME = "(0[0-9]|1[0-2]):[0-5][0-9] [AP]M( -+[0-9])?";
```

Comment:
Suppose the UTC date is 11/07/2017 while the GMT+8 date is 12/07/2017. This will cause a html assertion error although the GMT +8 date was correct. We want to avoid these false negative results. It would be good if the regex used is specific to the timezone so that we will be able to assert that the eventually replaced result is displayed in the correct time zone.